packages/openneuro-server/src/graphql/resolvers/user.ts

```diff
@@ -9,7 +9,7 @@ function isValidOrcid(orcid: string): boolean {
 export const user = (obj, { id }) => {
   if (isValidOrcid(id)) {
     return User.findOne({
-      $or: [{ "orcid": id }, { "providerId": id }],
+      $or: [{ "provider": "orcid", "providerId": id }],
```

Comment:
@thinknoack I changed this because I was seeing a different set of datasets for different users based on what step in user migration was complete. I think it makes sense to only match this on ORCID users going forward now that the migration is live.